test/docfx.Test/DocfxTest.cs

```diff
@@ -281,6 +326,22 @@ private static bool IsRequiredOutput(string name)
                 && fileName != "server-side-dependent-list.txt";
         }
 
+        private static bool IsContentFile(string path)
```

Comment:
This is neccessary and you cannot distinguish a content file rightly until you parse the config.

Reply:
It is to skip tests that only has a `docfx.yml` and `a.md`, running single file build on these tests isn't quite meaningful.